docs/csharp/language-reference/compiler-messages/cs0465.md

```diff
@@ -10,11 +10,11 @@ ms.assetid: 3d36faae-147f-4173-b164-af953fd86eea
 ---
 # Compiler Warning (level 1) CS0465
 
-Introducing a 'Finalize' method can interfere with destructor invocation. Did you intend to declare a destructor?  
+Introducing a 'Finalize' method can interfere with finalizer invocation. Did you intend to declare a finalizer?  
```

Comment:
Again, this is the message produced by the compiler. So, shouldn't be changed.

Reply:
Fixed.